target/scripts/startup/setup-stack.sh

```diff
@@ -73,14 +73,16 @@ function _setup_amavis
   if [[ ${ENABLE_AMAVIS} -eq 1 ]]
   then
     _log 'debug' 'Setting up Amavis'
+
     cat /etc/dms/postfix/master.d/postfix-amavis.cf >>/etc/postfix/master.cf
+    echo -e \
+      "\n# Additional option for filtering\ncontent_filter = smtp-amavis:[127.0.0.1]:10024" \
+      >>/etc/postfix/main.cf
```

Comment:
```suggestion
    # Additional option for filtering
    postconf 'content_filter = smtp-amavis:[127.0.0.1]:10024'
```

Reply:
Done in 63abdb8824ce2a73f16b66e100f71f0479967065